cmd/gke-gcloud-auth-plugin/main.go

```diff
@@ -12,6 +12,9 @@ import (
 
 var (
 	useApplicationDefaultCredentials = pflag.Bool("use_application_default_credentials", false, "Output is an ExecCredential filled with application default credentials.")
+	useEdgeCloud                     = pflag.Bool("use_edge_cloud", false, "Output is an ExecCredential for an edge cloud cluster.")
```

Comment:
Nit: Probably capitalize "Edge Cloud cluster" since it's a specific product.

Reply:
Done, thanks